org.mixedrealitytoolkit.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -328,7 +328,7 @@ public void StartPlacement()
             // selected and then immediately unselected. If two calls occur within the
             // double click timeout, then return to prevent an immediate object state switch.
             // Also, check that time is no 0 to allow for auto start functionality.
-            if (Time.time != 0 && (Time.time - lastTimeClicked) < DoubleClickTimeout)
+            if (Time.time != 0 && (Time.time - lastTimeClicked) < DoubleClickTimeout && !placementRequested)
```

Comment:
why not just move `lastTimeClicked = Time.time;` after the `startCalled` change (line 348).   That remove the need to reset placementRequested.   And maybe initialize lastTimeClicked to `float.MaxValue`

Also..IMO, resetting placementRequested makes less sense, given the variable name.  If a reset is preferred...a better name might be `placementRequestedAndPending`

Reply:
I agree to move the `lastTimeClicked = Time.time;`.

We can't initialize lastTimeClicked to `float.MaxValue`, otherwise `(Time.time - lastTimeClicked) < DoubleClickTimeout` will be `true`, thus the function returns and never starts the tap to place.